src/main/utils.ts

```diff
@@ -40,6 +40,10 @@ import type { HolochainManager } from './holochainManager';
 import type { LauncherEmitter } from './launcherEmitter';
 import { DEFAULT_APPS_DIRECTORY } from './paths';
 
+export const isMac = process.platform === 'darwin';
+export const isWindows = process.platform === 'win32';
+export const isLinux = process.platform === 'linux';
```

Comment:
maybe we can use whenever is necessary instead of defining new utility functions

```
import { platform } from '@electron-toolkit/utils';
```

Reply:
eg
```
platform.isWindows,
```

Reply:
Okay, changed.